cluster/get-kube-local.sh

```diff
@@ -55,8 +55,9 @@ function run {
 }
 
 # Creates a kubeconfig file for the kubelet.
-# Args: destination file path
+# Args: the IP address of the API server (e.g. "http://localhost:8080"), destination file path
 function create-kubelet-kubeconfig() {
+  local api_addr="${1}"
```

Comment:
this var isn't used (the api addr is hard coded to localhost:8080 on line 75). You should either use the var or comment that it is ignored.